sample-compose/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Templates
+
+- Our optimized Android templates used in our android projects
+
+## Setup
+
+- Clone the project
+- Run the project with Android Studio
+
+## Linter and static code analysis
+
+- Lint:
+
+```
+$ ./gradlew lint
+```
+
+Report is located at: `./app/build/reports/lint/`
+
+- Detekt
+
+```
+$ ./gradlew detekt
+```
+
+Report is located at: `./build/reports/detekt`
+
+## Testing
+
+- Run unit testing:
+
+```
+$ ./gradlew app:testStagingDebugUnitTest
+$ ./gradlew data:testDebugUnitTest
+$ ./gradlew domain:test
+```
+
+- Run unit testing with coverage:
+
+```
+$ ./gradlew jacocoTestReport
+```
+
+Report is located at: `./app/build/reports/jacoco/`
+
+## Build and deploy
+
+For `release` builds, we need to provide release keystore and signing properties:
+
+- Put the `release.keystore` file at root `config` folder.
+- Put keystore signing properties in [signing.properties](https://github.com/nimblehq/android-templates/blob/develop/template/signing.properties).
```

Comment:
```suggestion
- Put keystore signing properties in [signing.properties](https://github.com/nimblehq/android-templates/blob/develop/sample-compose/signing.properties).
```

Reply:
Hi @Tuubz Should we remove the link entirely as we did in [this PR](https://github.com/nimblehq/android-templates/pull/277/files#diff-3d032ada187f4df44d78585f6651d8dda1ea51ca3f742092a1570b01491fe6d0)?

Reply:
Yes, please 😊